omegatrade-app/frontend/src/app/services/validation.service.ts

```diff
@@ -0,0 +1,30 @@
+export class ValidationService {
+    static getValidatorErrorMessage(validatorName: string, validatorValue?: any): object {
+      const config = {
+        required: 'Required',
+        invalidCreditCard: 'Is invalid credit card number',
```

Comment:
nit: is this used anywhere?

Reply:
removed unused validation as it is not going to be used anywhere.